openage/game/main_cpp.pyx

```diff
@@ -37,6 +37,18 @@ def run_game(args, root_path):
         else:
             args_cpp.mods = vector[string]()
 
+        # window width
+        if args.width is not None:
+            args_cpp.width = args.width
+        else:
+            args_cpp.width = 1024
+
+        # window height
+        if args.height is not None:
+            args_cpp.height = args.height
+        else:
+            args_cpp.height = 768
```

Comment:
Instead of checking for `None`, you could also just set the `default` argument for the respective args to the default values.